todoman/ui.py

```diff
@@ -251,8 +259,19 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        :param datetime.datetime date: a datetime object
+        Returns date in the following format:
+        * if date == today or tomorrow: "Today" or "Tomorrow"
+        * else: return a string representing that date
+        * if no date is supplied, it returns empty_date
+        """
+        assert isinstance(date, datetime)
```

Comment:
Oh! So sorry, I completely forgot about it!

Should I also add a test case where it would check for the case when format_date receives ```None```